kyuubi-server/pom.xml

```diff
@@ -266,6 +266,13 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.kyuubi</groupId>
+            <artifactId>kyuubi-hive-beeline</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
I think you should exclude `*` from `kyuubi-hive-beeline` to prevent dependency transfer.